src/compas_fab/ghpython/reachability_artist.py

```diff
@@ -0,0 +1,85 @@
+from compas.geometry import Frame
+from compas.colors import ColorMap
+from compas.artists import PrimitiveArtist
+from ghpythonlib.treehelpers import list_to_tree
+
+
+class ReachabilityMapArtist(object):  # how base on GHArtist without error?
```

Comment:
This should really inherit from `GHArtist`, which errors were you getting?

Reply:
```
Runtime error (Exception): No artist is registered for this data type: <class 'compas_fab.robots.reachability_map.reachability_map.ReachabilityMap'> in this context: Grasshopper

Traceback:
  line 61, in _get_artist_cls, "C:\Users\rustr\AppData\Roaming\McNeel\Rhinoceros\7.0\scripts\compas\artists\artist.py"
  line 92, in __new__, "C:\Users\rustr\AppData\Roaming\McNeel\Rhinoceros\7.0\scripts\compas\artists\artist.py"
  line 5, in script
```